### ISSUE TITLE
[BUG] Error in Predict Percept

### ISSUE THREAD

**Issue by anvithaakkaraju** [pulse2percept/pulse2percept#151]
Reproduce the bug:
```
model = p2p.models.AxonMapModel()
model.build()
model.predict_percept(implant=ArgusII)
```

When I pass Argus II into the predict method, I get an error about left/right eye.

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-17-e310b8938853> in <module>()
----> 1 model.predict_percept(implant=ArgusII)

/content/p2psource/pulse2percept/models/axon_map.py in predict_percept(self, implant, t)
    340         if implant.eye != self.eye:
    341             raise ValueError(("The implant is in %s but the model was built "
--> 342                               "for %s.") % (implant.eye, self.eye))
    343         return super(AxonMapModel, self).predict_percept(implant, t=t)

ValueError: The implant is in <property object at 0x7f96acf818b8> but the model was built for RE.
```

**Comment by mbeyeler** [comment]
The error message is wrong - it should say something like "You must pass an instance of class ProsthesisSystem". The problem was that it should say `model.predict_percept(ArgusII())`, not `model.predict_percept(ArgusII)` (note the parentheses).

**Comment by mbeyeler** [comment]
Closed via #168
